### DISCUSSION TITLE
how use multisearch?

### DISCUSSION THREAD

**Discussion by numbereleven-a** [winsiderss/systeminformer#1995] [Questions / Answers] [answered]
![b91a39907769e5b448b6a16ca7a860fe](https://github.com/winsiderss/systeminformer/assets/139941473/5da76e8b-5ca9-4bc8-9f58-55d8ee2535b3)

**Comment by numbereleven-a** [comment]
2. and is it possible to write a process that does not need to be displayed?

**Comment by jxy-s** [comment, marked as answer]
That button symbolizes [regular expressions](https://en.wikipedia.org/wiki/Regular_expression). With that button enabled, in this case, you would want `tsl|pubg` to search for "tsl" or "pubg".

**Comment by numbereleven-a** [comment]
thanks

**Comment by numbereleven-a** [comment]
I can’t find the exception sign in regular expressions. :\

**Reply by jxy-s** [reply]
I guess that depends on what you're trying to do. It might not be working as you expect since the regex will match on any string that is displayed. An example of a regular expression to search for something doesn't contain something else is:
```regex
svchost\.exe -k (?!local|net)
```
This matches on strings that contain `svchost.exe -k ` but are not followed by `local` or `net`.

In contrast, something like this by itself will match on a bunch of cells because many of them don't contain it:
```regex
(?!svchost)
```

A way to augment this might be:
```regex
(?<!svchost)\.exe
```
In that example it would match any cells containing `.exe` but does not start with `svchost` which should filter out all `svchost.exe` processes.

**Reply by numbereleven-a** [reply]
(?<!svchost) \ .exe - its worked, thanks again
